### ISSUE TITLE
BWA-meth built-in reference genomes in *.org

### ISSUE THREAD

**Issue by gallardoalba** [galaxyproject/galaxy#13122]
A user in [GHelp](https://help.galaxyproject.org/t/no-option-available-to-select-the-reference-genome/7272) requested the hg38 genome in BWA-meth, required for the [DNA Methylation data analysis](https://training.galaxyproject.org/training-material/topics/epigenetics/tutorials/methylation-seq/tutorial.html) training.

**Comment by bgruening** [comment]
We have this on EU and I answered that to the user via mail. He has sent me a personal mail.

**Comment by gallardoalba** [comment]
> 



> We have this on EU and I answered that to the user via mail. He has sent me a personal mail.

Yes, he also wrote in Ghelp and I suggested him to use *eu. Just I don't know if it is necessary to include the references also in *org.

**Comment by bgruening** [comment]
@natefoo call. Nate you can also copy them from our server if you like.

**Comment by natefoo** [comment]
Yeah this would be good.
